src/config/lang/english.go

```diff
@@ -618,6 +619,11 @@ const (
 	AgentErrUnableTransform        = "unable to transform the provided request; see zarf http proxy logs for more details"
 )
 
+// src/internal/packager
+const (
+	PkgErrsetVariableMap = "unable to set the active variables: %w"
```

Comment:
```suggestion
	PkgErrSetVariableMap = "unable to set the active variables: %w"
```

Reply:
Also will leave it up to @Noxsios as to whether this should be here - the lang pattern we have should be reevaluated.